src/rules/indentRule.ts

```diff
@@ -33,13 +36,30 @@ export class Rule extends Lint.Rules.AbstractRule {
         optionsDescription: Lint.Utils.dedent`
             One of the following arguments must be provided:
 
-            * \`"spaces"\` enforces consistent spaces.
-            * \`"tabs"\` enforces consistent tabs.`,
+            * \`${OPTION_USE_SPACES}\` enforces consistent spaces.
+            * \`${OPTION_USE_TABS}\` enforces consistent tabs.
+
+            A second optional argument specifies indentation size, defaulting to 4:
+
+            * \`${OPTION_INDENT_SIZE_2.toString()}\` enforces 2 space indentation.
```

Comment:
Yeah that surprised me as well! I thought it wasn't necessary but TS is telling me it is:
![image](https://cloud.githubusercontent.com/assets/4172079/25891787/32ce8514-356a-11e7-94c0-3e9f5c12cb4d.png)